static/css/api_key_settings.css

```
@@ -1029,6 +1029,155 @@ input:checked + .slider:before {
+    right: 6px;
+    font-size: 14px;
+    font-weight: bold;
+    color: #40C5F1;
+}
+
+.gsv-voice-card .gsv-card-name {
+    font-size: 0.92em;
+    font-weight: 600;
+    color: #333;
+    margin-bottom: 2px;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)